samples/perf_hgemm.cpp

```diff
@@ -270,11 +270,11 @@ __device__ static inline void globalWriteD(float16_t* gAddrD,
                                            MfmaFragD const (&fragsD)[BLOCKS_X][BLOCKS_Y],
                                            uint32_t ldd)
 {
-    auto blockWriteOffsetA = make_coord2d(ROCWMMA_M, 0u);
-    auto blockStepX = get<1>(blockWriteOffsetA) * ldd + get<0>(blockWriteOffsetA);
+    auto readOffset = make_coord2d(ROCWMMA_M, 0u);
```

Comment:
Please take into account the function name when naming the locals.
This is a write function so it would be better named to writeOffset, as an example.